src/README.md

```diff
@@ -1,17 +1,22 @@
-Welcome to the source of ReasonReact!
+### Welcome to the source of ReasonReact
+
+We want to expose as minimum modules to the toplevel as possible, and at the same time want to map closely to the npm packages. So each module maps to a npm package, and each sub-module maps to a sub-module. For example: `react-dom` -> `ReactDOM` and `react-dom/server` -> `ReactDOM.Server`.
 
 Files overview:
 
 ## Bindings
```

Comment:
thanks for writing this up. it's very clear. 

you may want to check the rendering before merging, as the nested bullets aren't rendered as you'd expect.